src/libraries/System.Text.Json/tests/System.Text.Json.SourceGeneration.Tests/JsonSerializerContextTests.cs

```diff
@@ -56,7 +56,9 @@ static void AssertFieldNull(string fieldName, JsonSerializerOptions? optionsInst
                         Assert.NotNull(fieldInfo);
                         Assert.Null(fieldInfo.GetValue(optionsInstance));
                     }
-                }),
+
+                    return 0;
```

Comment:
This and the `new RemoteInvokeOptions() { ExpectedExitCode = 0 }` should be deleteable, right?